mchf-eclipse/drivers/ui/ui_driver.c

```diff
@@ -1570,6 +1581,17 @@ const BandGenInfo bandGenInfo[] =
 		{21450000, 21750000, "13m" },
 		{25670000, 26100000, "11m" },
 		{26965000, 27405000, "11m" },
+//[QBS]s
+		{27415000, 27595000, "555" },//[QBS] Triple Five
```

Comment:
We will not accept this, the UHSDR firmware is not a UK firmware so please do not put UK specific bandplan information in the official menu.

Reply:
Hi Guys, been really unwell this week, just starting to feel abit better today. Im going to have to tackle the commit 

suituation when im totally well. Yes i very much understand what you guys have said about what i submitted, I just need to 

learn how to only submit specific things to you guys. I think ive done well with the CPU UUID code as this acts as a unique 

serial number for the radio as its read only burned into the CPU at the factory. Ive now worked out how to keep my code 

inline with the main head fork, what i do is a pull request on my on fork when i see that there has been changes to the main 

head, i submit the changes in the head fork to my fork then accept them. So far so good :) if you guys would like the CPU 

UUID code ive pasted the two submits here the original then the one with the edited removed obsolete bits...  Thank you for 

being patient with me. I find it hard to take everything in with my health condition. But would very much like to contribute 

in some way.  I have a question would you guy be interested if i also added Elecraft K3 Cat control that is switchable in the menu system like the Yaesu Cat control code is, It would even be possible to have both enabled at the same time if wished. The reason for saying this is because there are some programs that are K3 compatible but not Yaesu compatible, There is a version of PowerSDR that works with the K3. Please find below the link to the two submits for the CPU UUID code
regards Richard 2E0DVK, ps i always tag my changes because it makes it very fast to find them in the code base as i just search [QBS] its the way me and a team worked back some 18 years or so ago i used to write mods for the Quake 2 computer game. I also wrote a full 2d/3d gaming engine back then, 

CPU UUID (Unique serial number)
https://github.com/quadrant2005/UHSDR/commit/372ae301d96c8f03f62586933964fc736e1d79e1
https://github.com/quadrant2005/UHSDR/commit/246603dab35bf5b7aade1cc57ad5dd8d490b49d9